radixdlt-java/src/main/java/com/radixdlt/client/application/RadixApplicationAPI.java

```diff
@@ -263,6 +264,7 @@ public static RadixApplicationAPI create(RadixIdentity identity) {
 			.addStatelessParticlesMapper(new CreateTokenToParticlesMapper())
 			.addStatelessParticlesMapper(new MintTokensActionMapper())
 			.addStatelessParticlesMapper(new UniqueIdToParticlesMapper())
```

Comment:
Similar to my comment on the `UniqueIdAction` this should also be `AddUniqueIdToParticlesMapper` or something to that effect